web/src/lib/domain-logic/filterFilings.ts

```diff
@@ -15,12 +15,16 @@ export const sortFilingsEarliestToLatest = (filings: TaxFiling[]): TaxFiling[] =
   });
 };
 
-export const deadlinesWithinAYear = (filings: TaxFiling[]): TaxFiling[] => {
+export const currentAndFutureDeadlines = (filings: TaxFiling[]): TaxFiling[] => {
```

Comment:
it's still within a year, though, I might add that to the function name

Reply:
also, "Upcoming" might be a good word to use instead of "currentAndFuture" to make this function name a little shorter and easier to read (as well as the other renamed variables like "sortedFilteredFilingsCurrentAndFutureMonths")